sorc/orog_mask_tools.fd/inland.fd/inland.F90

```diff
@@ -1,8 +1,16 @@
 !> @file
-!! This program creates the inland mask and writes it to the orography data files.
-!! 
+!! @brief Program and associated routines for creating inland mask.
 !! @author Ning Wang @date July 1, 2020
+
+!> This program creates the inland mask and writes it to the orography data files.
+!!
```

Comment:
I am getting doxygen warnings about these four @params because they are not part of the routine argument list.

Reply:
Interesting, I don't get those warnings  on my computer.

Can I document command line arguments? @edwardhartnett

Reply:
I am compiling on our Dell.  It uses doxygen 1.8.5

Reply:
1.8.5 is pretty old.

I would suggest we commit these changes and keep working on the rest of the warnings. We will circle back to this again if the warnings appear on the CI runs.